js-intermediate.js

```diff
@@ -93,7 +174,7 @@ var myCar = {
   }
 }
 
-
+console.log(myCar.specs.doors , myCar.specs.transmission)
 
```

Comment:
To use destructuring you can save the values as myCar.specs
const { doors, transmission } = myCar.specs
console.log(doors, transmission)